src/ztc/api/tests/test_statustype.py

```diff
@@ -43,7 +43,8 @@ def test_get_detail(self):
             datum_einde_geldigheid=date(2021, 2, 1),
             zaaktype=zaaktype,
             eigenschappen=[eigenschap],
-            doorlooptijd_status="P30D",
+            doorlooptijd_periode=900,
+            periode_eenheid="werkdagen",
```

Comment:
leuk dat concept van "werkdagen", maar hier kan je als client toch praktisch niets mee om te berekenen van een startdatum + (periode van 900 werkdagen) = ???

ik verwacht hier echt dat de server kan bepalen wat de werkdagen zijn, en dan deze doorlooptijd periode kan vertalen naar kalendardagen wat in ISO-8601 uitgedrukt kan worden. Straks moet voor durations elke client zelf deze berekening gaan doen want sowieso tot verschillende resultaten gaat leiden

Reply:
Ben het met je eens dat een duration veel logischer is maar ik ging uit van wat hier in stond: 
https://www.gemmaonline.nl/index.php/Imztc_2.2/doc/gegevensgroeptype/statustype.doorlooptijd_status

Dit blijkt niet te kloppen en ook blijkt dat deze ticket al gedaan was op de correcte manier (een DaysDurationField voor de doorlooptijd_status). Dus ik sluit deze PR.

Reply:
ik vind het een beetje makkelijk om te verwijzen naar ImZTC en niet meer kritisch na te denken over hoe dit gebruikt moet worden. Het was net met StUF-ZKN het probleem dat een letterlijke vertaling van informatiemodel naar XML-definities tot slechte bruikbaarheid leidt voor de daadwerkelijke functionaliteit.

Mag ik jullie verzoeken om aub niet de _gebruikers_ van de standaard uit het oog te verliezen? Dus actief nadenken hoe je dit als client moet gaan gebruiken.

Reply:
aangezien ik hier pas een maand zit was ik niet op de hoogte van dat de ImZTC soms niet 100% te volgen is (zoals mij wel is verteld). Maar prima.